code/modules/surgery/organs/subtypes/grey.dm

```diff
@@ -1,3 +1,17 @@
 /obj/item/organ/internal/liver/grey
 	alcohol_intensity = 1.6
 	species = "Grey"
+
+/obj/item/organ/internal/brain/grey
+	icon_state = "brain-x"
+	mmi_icon_state = "mmi_alien"
+	species = "Grey"
+
+/obj/item/organ/internal/brain/grey/insert(var/mob/living/carbon/M, var/special = 0)
+	..()
+	M.add_language("Psionic Communication")
```

Comment:
It's no big deal, I'm just being silly. You could yank out a Vulp brain and a grey brain, put the grey brain in the vulp then defib the vulp.